src/__init__.py

```diff
@@ -102,23 +88,30 @@ def _check_timepoints(t_ampa_ms, t_gaba_ms, tau_ampa, tau_gaba, t_eval_ms, I_amp
         # ampa has to be equal to or smaller than eval
         # for a range of ampa, if eval is within range, use linear/quadratic interpolation
         # if not, raise exception
+        interp_ampa = None
+
         for t in [np.min(t_eval_ms), np.max(t_eval_ms)]:
```

Comment:
you shouldn't be using a for loop; you can do these things with array operations